frontend/src/layout/navigation/navigationLogic.ts

```diff
@@ -1,14 +1,16 @@
+import dayjs from 'dayjs'
 import { kea } from 'kea'
-import api from 'lib/api'
-import { systemStatusLogic } from 'scenes/instance/SystemStatus/systemStatusLogic'
+import api from '../../lib/api'
+import { FEATURE_FLAGS } from '../../lib/constants'
+import { featureFlagLogic } from '../../lib/logic/featureFlagLogic'
+import { eventUsageLogic } from '../../lib/utils/eventUsageLogic'
+import { systemStatusLogic } from '../../scenes/instance/SystemStatus/systemStatusLogic'
+import { organizationLogic } from '../../scenes/organizationLogic'
+import { preflightLogic } from '../../scenes/PreflightCheck/logic'
+import { teamLogic } from '../../scenes/teamLogic'
+import { userLogic } from '../../scenes/userLogic'
+import { VersionType } from '../../types'
```

Comment:
@Twixes It's a nit, but I often see your IDE work against some other IDEs or configs in a `short<->relative` import renaming battle royale.

```diff
+import { teamLogic } from '../../scenes/teamLogic'
-import { teamLogic } from 'scenes/teamLogic'
```

Personally I'd use the shorter ones, if there's a defined alias (in tsconfig.json) in the path like `scenes/` or `lib/` or `~/`.

Reply:
I agree, it's just my VS Code set up doing not taking `tsconfig` into account 😅 I'll look into making it use the shorter paths

Reply:
Turns out for some reason I had this set to the non-default "relative" value:
![:O](https://user-images.githubusercontent.com/4550621/142622777-d077315d-1957-49c1-be48-d4e904032e34.png)
You can now sleep well knowing that my future PRs will have the shortest imports possible. 😄